nodes/stampzilla-server/models/connection.go

```diff
@@ -1,8 +1,12 @@
 package models
 
+import "github.com/olahol/melody"
+
 type Connection struct {
 	Type       string                 `json:"type"`
 	RemoteAddr string                 `json:"remoteAddr"`
 	NodeUuid   string                 `json:"nodeUuid,omitEmpty"`
 	Attributes map[string]interface{} `json:"attributes"`
+
+	Session *melody.Session `json:"-"`
```

Comment:
Is this used anywhere? searched for `.Session` in this PR and found nothing?

Reply:
I found a couple of places it was used in `webserver/webserver.go`
![stamp_2020-09-04_100406](https://user-images.githubusercontent.com/1146381/92280954-de7a1980-eefa-11ea-942e-1fa82f757187.png)
and
![stamp_2020-09-04_100349](https://user-images.githubusercontent.com/1146381/92280957-df12b000-eefa-11ea-8561-f74fb119542e.png)